packages/config/package.json

```diff
@@ -28,7 +28,6 @@
     "hapi-pino": "^8.5.0",
     "hapi-sentry": "^3.2.0",
     "io-ts": "^2.2.18",
-    "is-svg": "^4.3.2",
```

Comment:
did you run "yarn install" after this? just curious as there isn't yarn lock changes